src/components/IconToggle/IconToggle.driver.ts

```diff
@@ -0,0 +1,14 @@
+import { checkboxDriverFactory as coreCheckboxDriverFactory } from 'wix-ui-core/drivers/vanilla';
+import { errorMessageWrapperDriverFactory } from '../../baseComponents/ErrorMessageWrapper/ErrorMessageWrapper.driver';
```

Comment:
I'm assuming this is still work in progress, but don't forget to change this to the correct component